DQM/TrackingMonitorSource/plugins/StandaloneTrackMonitor.cc

```diff
@@ -527,36 +1248,160 @@ void StandaloneTrackMonitor::analyze(edm::Event const& iEvent, edm::EventSetup c
       double pt = track.pt();
 
       const reco::HitPattern& hitp = track.hitPattern();
-      double nValidTrackerHits = hitp.numberOfValidTrackerHits();
-      nHitsVsEtaH_->Fill(eta, nValidTrackerHits);
-      nHitsVsCosThetaH_->Fill(std::cos(theta), nValidTrackerHits);
-      nHitsVsPhiH_->Fill(phi, nValidTrackerHits);
-      nHitsVspTH_->Fill(pt, nValidTrackerHits);
-      nHitsVsnVtxH_->Fill(vertexColl->size(), nValidTrackerHits);
+      double nAllHits = hitp.numberOfAllHits(reco::HitPattern::TRACK_HITS);
+      double nAllTrackerHits = hitp.numberOfAllTrackerHits(reco::HitPattern::TRACK_HITS);
+
+      double trackdeltaR = 100000000000;
```

Comment:
```suggestion
      double trackdeltaR = std::numeric_limits<double>::max(); 
```
might be more elegant?

Reply:
Is there a reason why this was not addressed?